WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -126,4 +126,10 @@ def wca_date_range(from_date, to_date, options={})
     options[:separator] = '-'
     date_range(from_date, to_date, options)
   end
+
+  def alert(type, content=nil, note: false, &block)
+    content = capture(&block) if block_given?
+    content.prepend("<strong>Note:</strong> ") if note
+    content_tag :div, content.html_safe, class: "alert alert-#{type}"
```

Comment:
Hmm... I feel like if content was passed in as a parameter (not via a block), maybe we actually *don't* want to call `.html_safe` on it?